packages/ag-charts-community/src/scene/util/sector.ts

```diff
@@ -14,6 +14,40 @@ interface LineCoordinates {
     end: { x: number; y: number };
 }
 
+export function sectorBox({ startAngle, endAngle, innerRadius, outerRadius }: SectorBoundaries) {
```

Comment:
This new function seems to do the same thing as `computeSectorFocusBounds`:
https://github.com/ag-grid/ag-charts/blob/dbd45c5f9b8ad7ec11c413482e232ab87cc6ef83/packages/ag-charts-community/src/chart/series/polar/pieUtil.ts#L168

If so, it would be worth joining these two functions to avoid code duplication.